src/test/java/StringTest.java

```diff
@@ -0,0 +1,34 @@
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class StringTest {
+
+    @Test
+    void split() {
+        String[] result1 = "1,2".split(",");
+        String[] result2 = "1".split(",");
+
+        assertThat(result1).containsExactly("1", "2");
+        assertThat(result2).containsExactly("1");
+    }
+
+    @Test
+
+    void substring() {
```

Comment:
공백 라인이 없으면 가독성이 더 좋아질 것 같습니다 😃
```suggestion
    @Test
    void substring() {
```

Reply:
확인이 늦었네요ㅠ 피드백 감사합니다! 다음단계도 얼른 완성해서 PR 요청 드리겠습니다.